### ISSUE TITLE
Include a Full-Featured systemd

### ISSUE THREAD

**Issue by naftulikay** [naftulikay/docker-xenial-vm#5]
Many things aren't working like `timedatectl` which work in CentOS 7 systemd. Remove [these lines](https://github.com/naftulikay/docker-xenial-vm/blob/develop/Dockerfile#L16-L25):

```
    # ...
    && ( cd /lib/systemd/system/sysinit.target.wants/; for i in *; do \
      [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; \
    done ); \
    rm -f /lib/systemd/system/multi-user.target.wants/*; \
    rm -f /etc/systemd/system/*.wants/*; \
    rm -f /lib/systemd/system/local-fs.target.wants/*; \
    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
    rm -f /lib/systemd/system/basic.target.wants/*; \
    rm -f /lib/systemd/system/anaconda.target.wants/*; \
```

All systemd services must not be removed and must actually be manually installed such that the right things are available.

**Comment by naftulikay** [comment]
Symptom, `timedatectl` doesn't work on Xenial but does on CentOS 7.

**Comment by naftulikay** [comment]
Closed by #6.
